core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -113,6 +114,7 @@ object TopicCommand extends Logging {
     def printDescription(): Unit = {
       val configsAsString = config.entries.asScala.filter(!_.isDefault).map { ce => s"${ce.name}=${ce.value}" }.mkString(",")
       print(s"Topic: $topic")
+      print(s"\tTopicId: ${if(topicId == Uuid.ZERO_UUID) "" else topicId}")
```

Comment:
Do we want to have a field TopicId that is empty? Would it make sense to include the Zero Uuid or not print a topic ID label at all?